release-notes/release-notes56.md

```diff
@@ -235,22 +195,10 @@ For information on:
 
 ## Security fixes
 
-{{site.productname}} 5.6 provides fixes for the following security issues:
-
-- changelog
-
 PowerPaste 5.4.0 provides fixes for the following security issues:
 
-* Fixed a Cross-Site Scripting (XSS) vulnerability where internal HTML content wasn't sanitized in some cases.
-* Fixed a Cross-Site Scripting (XSS) vulnerability where specific HTML comments weren't sanitized in some cases.
-
-## Deprecated features
-
-The following features have been deprecated with the release of {{site.productname}} 5.6:
-
-- [](#).
-
-### The...
+- Fixed a Cross-Site Scripting (XSS) vulnerability where internal HTML content wasn't sanitized in some cases.
```

Comment:
We'll need to add the TinyMCE security fix here as well at some point (atm it's in the general bug fixes section):
> Fixed a security issue where URLs in attributes weren't correctly sanitized.

Reply:
going to handle this in the next PR once I merge this one. I have a few changes to make similar to this one, and I'm not going to make you all re-review this PR for them. :slightly_smiling_face: